### PR TITLE
Cleanup team data

### PR DESCRIPTION
**Pull request by pgfisico**
Continuation of the changes from #138

- Remove images that were removed
- Rename and move image files
- Rename and alphabetize imports
- Remove extra whitespace
- Remove commented code

See #34, #35, #36

### CODE REVIEW

File: src/data/team.js [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
-import team_sarah from '../images/team/team/sarah.jpg'
-import team_sraddha_2 from '../images/team/team/sraddha.2.jpg'
-import team_tomoe from '../images/team/team/tomoe.jpg'
+
 import team_amelie from '../images/team/team/AmelieProfile.jpg'
 import team_celina from '../images/team/team/CelinaProfile.jpg'
 import team_elena from '../images/team/team/ElenaProfile.jpg'
+import team_emma from '../images/team/team/emma.jpg'
+// TODO Should this be JuliaProfile.jpg ?
+import team_julia from '../images/team/team/julia.jpg'
```
Comment (pgfisico) [inline src/data/team.js:7]: @egyajure Was this supposed to be updated to refer to the image that was added in the previous PR?

https://github.com/intimitrons4604/website/commit/fc32d39cbe099e402e0f75ce6de9a3f454c7539c#diff-73f30f0662fe4b41eb1710f0db88784d9df26a96ef6698dcc915f515154c5ff7

Comment (egyajure) [reply src/data/team.js:6]: @pgfisico Oops, yes I meant to update that photo with JuliaProfile

**Review by egyajure — APPROVED** [review]
Looks good, thanks for cleaning things up :)

File: src/data/team.js (hunk 1 above)
Comment (pgfisico) [reply src/data/team.js]: Updated to the new one and removed the old one

**Pull request merged**